src/Sylius/Bundle/ApiBundle/Resources/config/serialization/ProductVariantTranslation.xml

```diff
@@ -8,18 +8,21 @@
         <attribute name="id">
             <group>shop:order:account:read</group>
             <group>admin:order:read</group>
-            <group>admin:product_variant:read</group>
```

Comment:
Why are you removing an id only from admin context?

Reply:
I left the rest because I didn't want to put too many changes in this PR. Removing the ID completely would also affect orders when this PR is about ProductVariant